cmd/ssh-chat/cmd.go

```diff
@@ -38,6 +38,7 @@ type Options struct {
 	Version    bool     `long:"version" description:"Print version and exit."`
 	Allowlist  string   `long:"allowlist" description:"Optional file of public keys who are allowed to connect."`
 	Whitelist  string   `long:"whitelist" dexcription:"Old name for allowlist option"`
+	Keynames   bool     `long:"keynames" description:"Reserve usernames by appending them (after a space) to their keys in the admin or allowlist files."`
```

Comment:
Maybe `ReserveNames` and `reserve-names`?

Reply:
`--reserve-names` does sound better - it says more directly what it does. `--keynames` says more about how it's done. I'll change that.